.drone.yml

```diff
@@ -730,6 +730,180 @@ trigger:
   - refs/heads/release*
   - refs/heads/develop*
 
+---
+kind: pipeline
+name: acceptance-webUIPrivateLinks
```

Comment:
I heard there was a thing called "starlark" and is supposed to generate a lot of the below code ?

@individual-it @phil-davis if this is not in place we should do so with higher priority next sprint

Reply:
yes, I have implemented it for most of the app repos - getting near to finishing them all now.
I can also do for Phoenix if you like. Please raise an issue in Phoenix and assign me, then I won't forget.

Reply:
https://github.com/owncloud/phoenix/issues/2074